lua/telescope/_extensions/coc.lua

```diff
@@ -399,7 +399,7 @@ end
 
 local workspace_diagnostics = function(opts)
   opts = utils.get_default(opts, {})
-  opts.hide_filename = utils.get_default(opts.hide_filename, false)
+  opts.path_display = utils.get_default(opts.path_display, "hidden")
```

Comment:
For workspace diagnostics, it's better to show file path because they're from different file.